landing_page/spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+    factory :user do
+      first_name { "John" }
```

Comment:
Have you heard of the faker gem? Consider if a test was to create two users via the factory, they both would have the same data at the moment. Faker will generate unique fields for your data so that this doesn't happen. Maybe this change can be made in a separate PR , what do you think @alvarocavalcanti ?

Reply:
Yeah I've heard of the Faker gem, I've used it before when I was playing around with creating a blog post and it was quite interesting to use

Reply:
Although I like the usage of `Faker` maybe we don't have a use case for it for the moment.

But a scenario in which using it would be required would be to render a list of articles or users, at least two, and assert that they are unique.